### PR TITLE
Add a Util to get the installed admin app version code

### PR DESCRIPTION
**Pull request by johnmanagexr**
To be used for compatibility checks

### CODE REVIEW

File: Assets/MXR.SDK/Plugins/Android/NativeUtils.java [hunk 1]
```diff
@@ -194,4 +194,29 @@ public String getInstalledAdminAppPackageName() {
...
+        List<PackageInfo> packages = pm.getInstalledPackages(0);
+
+        String adminAppPackageName = getInstalledAdminAppPackageName();
+        if (adminAppPackageName == null) return null;
+
+        for (PackageInfo packageInfo : packages) {
+            if (!packageInfo.packageName.equals(adminAppPackageName)) continue;
+
+            int versionCode;
+            // The default versionCode field is deprecated on versions > P.
```
Comment (Aleyum) [inline Assets/MXR.SDK/Plugins/Android/NativeUtils.java:208]: What is meant by `P` ?

File: Assets/MXR.SDK/Plugins/Android/NativeUtils.java [hunk 2]
```diff
@@ -194,4 +194,29 @@ public String getInstalledAdminAppPackageName() {
         return null;
     }
 
+    public int getInstalledAdminAppVersionCode() {
```
Comment (Aleyum) [inline Assets/MXR.SDK/Plugins/Android/NativeUtils.java:197]: Is there a reason why this function begins with a lower case and the one below, `GetAdminAppVersionCode` begins with an upper case ?

**Review by Aleyum — COMMENTED** [review]
LGTM, just nits.

File: Assets/MXR.SDK/Plugins/Android/NativeUtils.java (hunk 2 above)
Comment (johnmanagexr) [reply Assets/MXR.SDK/Plugins/Android/NativeUtils.java:197]: One of these is Java, and one of these is C#! These are just different norms based on languages. Comparing file to file (and language to language) these are consistent

File: Assets/MXR.SDK/Plugins/Android/NativeUtils.java (hunk 1 above)
Comment (johnmanagexr) [reply Assets/MXR.SDK/Plugins/Android/NativeUtils.java:208]: This is an Android constant for version Pie, which is SDK 28. This is common practice in Android land, which we're in with Java plugins.

Comment (Aleyum) [reply Assets/MXR.SDK/Plugins/Android/NativeUtils.java]: Cool!  I was wondering was it P for product, SGTM!

**Review by Aleyum — APPROVED** [review]

**Review by adrenak — APPROVED** [review]

**Pull request merged**